docs/paper/dev/api/obsolete-migration/material.md

```diff
@@ -0,0 +1,70 @@
+---
+slug: /dev/material
+---
+
+# Material Migration Guide
+
+Material was originally built when every single item had a corresponding block.
```

Comment:
I don't believe this line is true? There were numerous items that did not have a block equivalent at that point in time. stick, flint, coal, string, most of the non-cake food items, etc